doc/lua_api.txt

```diff
@@ -1573,8 +1570,7 @@ Exact pointing location (currently only `Raycast` supports these fields):
   from 1).
 * `pointed_thing.intersection_normal`: Unit vector, points outwards of the
   selected selection box. This specifies which face is pointed at.
-  Is a null vector `{x = 0, y = 0, z = 0}` when the pointer is inside the
-  selection box.
+  Is a null vector `(0, 0, 0)` when the pointer is inside the selection box.
```

Comment:
I don't like this because it's no longer valid Lua. What about `vector.zero()`?

Reply:
Changed to `vector.zero()`.